.github/workflows/security_scan.yaml

```diff
@@ -0,0 +1,83 @@
+name: Security Scan
```

Comment:
Is there any reason for not defining this workflow as part of the operator workflows? It would be simpler to reuse by other repositories

Reply:
I think it makes sense :)